src/tests/tests.cpp

```diff
@@ -1,6 +0,0 @@
-#include <gtest/gtest.h>
```

Comment:
I am assuming that test can be set to provide the `main()` function implementation? I would still like to have some comment on why this is (can be) removed.

Reply:
Yep, gtest provides its own `main()`. Only reason to define your own is if you want to have command line args or something like that.

Reply:
Does this mean that the standard gtest command line options (e.g. `--gtest_filter`) does not work anymore? If so, then I am not very happy with that change.

Reply:
No they continue to work as before.

Reply:
@SkyToGround Here is the main it adds: https://github.com/google/googletest/blob/master/googletest/src/gtest_main.cc
It is identical to what we had, except an additional `printf`.

Reply:
Oh, ok then.